lib/config/go_routes/go_router.dart

```diff
@@ -418,12 +418,15 @@ abstract class AppRoutes {
             GoRoute(
               path: ':roomid',
               pageBuilder: (context, state) {
+                final emptyMapArgument = {};
                 if (state.extra is ChatRouterInputArgument) {
                   final extra = state.extra as ChatRouterInputArgument;
                   switch (extra.type) {
                     case ChatRouterInputArgumentType.draft:
                       if (extra.data is String?) {
                         return CupertinoPage(
+                          arguments: emptyMapArgument,
```

Comment:
<img width="1666" alt="Screenshot 2024-02-21 at 14 45 36" src="https://github.com/linagora/twake-on-matrix/assets/99852347/bc61685d-de4a-4fae-9642-88a2733d19a7">


IMO: `arguments` is nullable,  no need